scripts/utils/simulateTimelockProposal.ts

```diff
@@ -1,14 +1,14 @@
 import { ethers } from 'hardhat';
-import {
-  MainnetContracts,
-  NamedAddresses,
-  ProposalDescription,
-  TemplatedProposalDescription
-} from '@custom-types/types';
+import { MainnetContracts, NamedAddresses, TemplatedProposalDescription } from '@custom-types/types';
 import { OptimisticTimelock } from '@custom-types/contracts';
 import { getImpersonatedSigner, time } from '@test/helpers';
 import { Contract } from '@ethersproject/contracts';
 import { forceEth } from '@test/integration/setup/utils';
+import { TRIBAL_COUNCIL_POD_ID } from '@protocol/optimisticGovernance';
+
+type PodConfig = {
```

Comment:
Would be better to import this since you're using it in multiple places

Reply:
Yep, this is WIP